docs/release-process.md

```diff
@@ -0,0 +1,175 @@
+inspect the `Dockerfile` and purge or explicitly pull each base image.
+
+Run `make image-build`.
+
+Verify that the tag was properly recorded by the binary in the container.
+* Run `podman run --rm -it quay.io/samba.org/samba-operator:latest`
+* The operator will fail to start because it is not running in kubernetes.
+  However, all you need to verify is that the first log line, with
+  "Initializing Manager", contains JSON values that have the correct tag in the
+  "Version" field.
+
+
+For the image that was just built, apply a temporary pre-release tag
+to it. Example:
+```
+podman tag quay.io/samba.org/samba-operator:{latest,v0.5pre1}
+```
+
+Log into quay.io.  Push the images to quay.io using the temporary tag. Example:
+```
+podman push quay.io/samba.org/samba-operator:{latest,v0.5pre1}
+```
+
+Wait for the security scan to complete. There shouldn't be any issues if you
+properly updated the base images before building. If there are issues and you
+are sure you used the newest base images, check the base images on quay.io and
+make sure that the number of issues are identical. The security scan can take
+some time, while it runs you may want to do other things.
+
+
+## GitHub Release
+
+When you are satisfied that the tagged version is suitable for release, you
+can push the tag to the public repo:
+```
+git push --follow-tags
+```
+
+Draft a new set of release notes. Select the recently pushed tag. Start with
+the auto-generated release notes from GitHub (activate the `Generate release
+notes` button/link). Add an introductory section (see previous notes for an
+example). Add a "Highlights" section if there are any notable features or fixes
+in the release. The Highlights section can be skipped if the content of the
+release is unremarkable (e.g. few changes occurred since the previous release).
+
+Each release provides a YAML file that can be used to deploy the operator
+on a Kubernetes cluster. Generate this file by running:
+```
+# replace example version with correct version number
+kustomize build config/default/ > samba-operator-v0.5-default.yaml
+```
+
+Prepare a "downloads" section for the release notes.
+Use the following partial snippet as an example:
+```
+## Download
+
+The samba-operator image can be acquired from the quay.io image registry:
+
+* By tag: quay.io/samba.org/samba-operator:v0.5
+* By digest: quay.io/samba.org/samba-operator@sha256:040307f53c3f3fd6a5935306f9898858b6c46b7e9c2ae46244e79c2bc42fef0d
+
+### Deploying the operator
+
+This operator can be deployed using the example file samba-operator-v0.5-default.yaml file, attached to this release. Example:
+
+kubectl apply -f samba-operator-v0.5-default.yaml
+
+This is equivalent to checking out the v0.5 tag from the git repository and using the default configuration. Example:
+
+git clone -b v0.5 https://github.com/samba-in-kubernetes/samba-operator
+cd samba-operator
+kubectl apply -k config/default
+
+```
+... using the image that was pushed earlier
```

Comment:
Above sentence is missing some context. Is this the right place?

Reply:
I merged it into a new sentance in the paragraph before the markdown block example. Does it seem better now?

Reply:
👍🏻